packs/data/journals.db/archetypes.json

```diff
@@ -374,8 +374,9 @@
casting]{Basic Cathartic Spellcasting} Feat 4</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p>You gain the basic spellcasting benefits. Each time you gain a spell slot of a new level from this archetype, add a spell of that spell level from your chosen tradition to your repertoire-either a common spell or another spell you've learned or discovered.</p><h2>@UUID[Compendium.pf2e.feats-srd.Cathartic Focus Spell]{Cathartic Focus Spell} Feat 4</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p>You learn the focus spell listed under your emotion's entry. You can cast it only while in emotional fervor. If you don't already have one, you gain a focus pool of 1 Focus Point, which you can Refocus while you use @UUID[Compendium.pf2e.actionspf2e.Settle Emotions]{Settle Emotions}.</p><h2>@UUID[Compendium.pf2e.feats-srd.Work Yourself Up]{Work Yourself Up} Feat 8</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p><strong>Requirements</strong> You have a reaction available and aren't prevented from using @UUID[Compendium.pf2e.actionspf2e.Catharsis]{Catharsis}.</p><hr><p>You concentrate on your mental state, tapping into that raw emotional energy. Take your Catharsis reaction, causing the catharsis activation and entering your emotional fervor as normal. The fervor lasts only until the end of your turn. At the end of your emotional fervor, you experience the effects of your emotional fallout, as normal.</p><h2>@UUID[Compendium.pf2e.feats-srd.Expert Cathartic Spellcasting]{Expert Cathartic Spellcasting} Feat 12</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Basic Cathartic Spellcasting]{Basic Cathartic Spellcasting}</p><hr><p>You gain the expert spellcasting benefits.</p><h2>@UUID[Compendium.pf2e.feats-srd.Infectious Emotions]{Infectious Emotions} Feat 12</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p>When you use @UUID[Compendium.pf2e.actionspf2e.Catharsis]{Catharsis}, one ally within 30 feet gains the catharsis activation benefits in addition to you. If these benefits require an emotional focus, they use your emotional focus as they're experiencing an empathic imprint of your emotions. Depending on the activation benefit, it might be impossible to grant it to an ally; for instance, pride could only have an effect for an ally if both you and the ally critically failed the same saving throw against the same effect.</p><h2>@UUID[Compendium.pf2e.feats-srd.Master Cathartic Spellcasting]{Master Cathartic Spellcasting} Feat 18</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Expert Cathartic Spellcasting]{Expert Cathartic Spellcasting}</p><hr><p>You gain the master spellcasting benefits.</p>",
```

Comment:
```suggestion
                "content": "<p><strong>Rarity</strong> Uncommon</p><hr><p>Emotions are powerful, and you can tap into that power to enhance your magic for a time. You learn to harness a particular emotional state under the right conditions, but be warned: emotions are volatile and destructive if left unchecked. The experience often leaves you reeling from the unhindered emotion, requiring a rest or distraction before tapping into it again.</p><section class=\"info\"><h4>Emotional Focus</h4><p>Some emotional states, such as love or hatred, require the caster to focus on a specific individual called their emotional focus. Your emotional focus can change over time with you choosing a new focus after an important part of your story is resolved. Work with your GM to determine when this change happens. An emotional focus doesn't get a say in the matter; the cathartic spellcaster's emotions determine the bond. However, making another PC your emotional focus could result in an uncomfortable social dynamic. You should check with that PC's player in advance to see if such a link is acceptable.</p></section><h2>@UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication} Feat 2</h2><p><strong>Prerequisites</strong> Charisma 14 or ability to cast spells from spell slotes</p><hr><p>You've learned to harness a particular emotion and mix it into your magic. Choose an emotion from the Emotional States section to be your catharsis emotion.</p><p>If you don't already cast spells from spell slots, you learn to cast spontaneous spells and gain the Cast a Spell activity. You gain a spell repertoire with one cantrip of your choice, from a spell list of your choice. You choose this cantrip from the common spells on your chosen spell list or from other spells to which you have access on that list. You're trained in spell attack rolls and spell DCs for that tradition. Your key spellcasting ability for these spells is Charisma.</p><p>If you can already cast spells from spell slots, you learn one additional cantrip from your spellcasting tradition. If you're a prepared caster, you can prepare this spell in addition to your usual cantrips per day; if you're a spontaneous caster, you add this cantrip to your spell repertoire.</p><p>You gain the @UUID[Compendium.pf2e.actionspf2e.Catharsis]{Catharsis} reaction and the @UUID[Compendium.pf2e.actionspf2e.Settle Emotions]{Settle Emotions} activity.</p><p><strong>Emotional States:</strong></p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Anger)]{Catharsis Emotion (Anger)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Awe)]{Catharsis Emotion (Awe)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Dedication)]{Catharsis Emotion (Dedication)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Fear)]{Catharsis Emotion (Fear)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Hatred)]{Catharsis Emotion (Hatred)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Joy)]{Catharsis Emotion (Joy)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Love)]{Catharsis Emotion (Love)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Misery)]{Catharsis Emotion (Misery)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Pride)]{Catharsis Emotion (Pride)}</p><p>@UUID[Compendium.pf2e.classfeatures.Catharsis Emotion (Remorse)]{Catharsis Emotion (Remorse)}</p><hr><p><strong>Special</strong> You can't select another dedication feat until you've gained two other feats from the cathartic mage archetype.</p><h2>@UUID[Compendium.pf2e.feats-srd.Basic Cathartic Spellcasting]{Basic Cathartic Spellcasting} Feat 4</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p>You gain the basic spellcasting benefits. Each time you gain a spell slot of a new level from this archetype, add a spell of that spell level from your chosen tradition to your repertoire-either a common spell or another spell you've learned or discovered.</p><h2>@UUID[Compendium.pf2e.feats-srd.Cathartic Focus Spell]{Cathartic Focus Spell} Feat 4</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p>You learn the focus spell listed under your emotion's entry. You can cast it only while in emotional fervor. If you don't already have one, you gain a focus pool of 1 Focus Point, which you can Refocus while you use @UUID[Compendium.pf2e.actionspf2e.Settle Emotions]{Settle Emotions}.</p><h2>@UUID[Compendium.pf2e.feats-srd.Work Yourself Up]{Work Yourself Up} Feat 8</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p><strong>Requirements</strong> You have a reaction available and aren't prevented from using @UUID[Compendium.pf2e.actionspf2e.Catharsis]{Catharsis}.</p><hr><p>You concentrate on your mental state, tapping into that raw emotional energy. Take your Catharsis reaction, causing the catharsis activation and entering your emotional fervor as normal. The fervor lasts only until the end of your turn. At the end of your emotional fervor, you experience the effects of your emotional fallout, as normal.</p><h2>@UUID[Compendium.pf2e.feats-srd.Expert Cathartic Spellcasting]{Expert Cathartic Spellcasting} Feat 12</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Basic Cathartic Spellcasting]{Basic Cathartic Spellcasting}</p><hr><p>You gain the expert spellcasting benefits.</p><h2>@UUID[Compendium.pf2e.feats-srd.Infectious Emotions]{Infectious Emotions} Feat 12</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Cathartic Mage Dedication]{Cathartic Mage Dedication}</p><hr><p>When you use @UUID[Compendium.pf2e.actionspf2e.Catharsis]{Catharsis}, one ally within 30 feet gains the catharsis activation benefits in addition to you. If these benefits require an emotional focus, they use your emotional focus as they're experiencing an empathic imprint of your emotions. Depending on the activation benefit, it might be impossible to grant it to an ally; for instance, pride could only have an effect for an ally if both you and the ally critically failed the same saving throw against the same effect.</p><h2>@UUID[Compendium.pf2e.feats-srd.Master Cathartic Spellcasting]{Master Cathartic Spellcasting} Feat 18</h2><p><strong>Prerequisites</strong> @UUID[Compendium.pf2e.feats-srd.Expert Cathartic Spellcasting]{Expert Cathartic Spellcasting}</p><hr><p>You gain the master spellcasting benefits.</p>",
```